src/TagsInput/components/InputField.js

```diff
@@ -77,47 +97,49 @@ class InputField extends React.PureComponent<Props, State> {
       forwardRef,
       onFocus,
       onBlur,
-      autoFocus,
+      onKeyPress,
     } = this.props;
     const { componentWidth } = this.state;
-    const dynamicStyle = StyleSheet.create({
-      common: {
-        minWidth,
-        maxWidth,
-        fontSize,
-      },
+
+    const dynamicStyle = {
+      common: { fontSize },
       input: {
         width: componentWidth,
       },
-    });
+    };
+
+    const sizerValue = value || (placeholder || '');
```

Comment:
What do you mean by `sizer`?

Reply:
In my thought, I call like that, this Text helper component (line 128), which is used for width computation for TextInput.